motion/p5js/connected-elements/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Marius Claßen
+author-url: https://github.com/mariusclassen
+title: Connected Elements
```

Comment:
Please use a different title for your sketch. connected elements is to general and is also the name of a law.  This means you should change the folder name and the title and the permalink